### PR TITLE
feat: add configurable filament locale

### PR DESCRIPTION
**Pull request by FurkanGM**
This pr for use specific locale on filament dashboard.

### CODE REVIEW

File: packages/admin/src/FilamentManager.php [hunk 1]
```diff
@@ -144,7 +144,7 @@ public function registerStyles(array $styles): void
         $this->styles = array_merge($this->styles, $styles);
     }
 
-    public function registerTheme(string | Htmlable | null $theme): void
```
Comment (danharrin) [inline packages/admin/src/FilamentManager.php:147]: Please remove all formatting changes from this PR

**Review by danharrin — CHANGES_REQUESTED** [review]

File: packages/admin/src/FilamentManager.php (hunk 1 above)
Comment (furkangzk) [reply packages/admin/src/FilamentManager.php]: done?

Comment (FurkanGM) [reply packages/admin/src/FilamentManager.php]: done?

**Comment by FurkanGM** [comment]
can you add config description?

File: packages/admin/config/filament.php [hunk 2]
```diff
@@ -27,6 +27,16 @@
...
 
+    /*
+    |--------------------------------------------------------------------------
+    | Filament Locale
+    |--------------------------------------------------------------------------
+    |
+    | ...
+    |
+    */
+    'locale' => null,
```
Comment (zepfietje) [inline packages/admin/config/filament.php:38]:
```suggestion

    'locale' => null,
```

**Review by zepfietje — CHANGES_REQUESTED** [review]

File: packages/admin/config/filament.php (hunk 2 above)
Comment (FurkanGM) [reply packages/admin/config/filament.php:39]: done

**Comment by FurkanGM** [comment]
> This PR doesn't have any effect on the actual used locale, apart from the `locale_has_pluralization` function?

Yes. But we can set app locale by filament::getLocale in provider

**Comment by pxlrbt** [comment]
That's what I would expect if I set the locale. What's the use of this PR otherwise? 🤔

**Comment by zepfietje** [comment]
@pxlrbt, this is exactly what I questioned as well. I've discussed it with @danharrin and we've decided to fix the use case of this PR in a different way. Dan has just opened a new pull request with that implementation #3155.

**Pull request closed without merging**